SPECS/containerized-data-importer/containerized-data-importer.spec

```diff
@@ -172,11 +197,15 @@ install -m 0644 _out/manifests/release/cdi-cr.yaml %{buildroot}%{_datadir}/cdi/m
 %license LICENSE
 %doc README.md
 %{_bindir}/virt-cdi-importer
+%{_bindir}/tools/cdi-containerimage-server
+%{_bindir}/tools/cdi-image-size-detection
+%{_bindir}/tools/cdi-source-update-poller
```

Comment:
Is there a reason we're putting these in a unique `tools` folder, and not putting them in `/usr/bin` directly? Do we not want them to be in the default `$PATH`? As far as I know, we don't currently ship subdirectories under `/usr/bin`, and `/usr/bin/tools` doesn't really mean anything, semantically.

For what it's worth- [openSUSE just puts these under `/usr/bin`.](https://build.opensuse.org/package/view_file/openSUSE:Factory/containerized-data-importer/containerized-data-importer.spec?expand=1)

Reply:
If you see [this file](https://github.com/kubevirt/containerized-data-importer/blob/main/cmd/cdi-importer/BUILD.bazel), the cdi-containerimage-server is under tools directory.  But [this source file](https://github.com/kubevirt/containerized-data-importer/blob/46c6aa994aec9267532e56d84dfa4deca5d0aefe/pkg/controller/import-controller.go) expects it in the /usr/bin/ directory.  I followed the former and included in the tools.  But you could be right here - it is prudent to include under /usr/bin.  I'll change that.